.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - ci # CHANGEME
+  pull_request:
+    branches:
+      - ci # CHANGEME
+
```

Comment:
We probably want to remove it running on pull_requests once continuous deployment is running. Its fine for now since CD does not exist but probably worth thinking about.

Reply:
I think we should have CI running on pull requests so that we know there are no linting errors and tests pass. Do you think we could have a seperate CD workflow that only runs on pushes to `main`?